### ISSUE TITLE
add zremrangebyrank

### ISSUE THREAD

**Issue by penghap** [tj/node-ratelimiter#46]
add `zremrangebyrank` after  [here](https://github.com/tj/node-ratelimiter/blob/2a4f3a22b9075c343b4cb4e213e58b948d004c2b/index.js#L72) to remove some useless data information.

```js
db.multi()
    .zremrangebyscore([key, 0, start])
    .zcard([key])
    .zadd([key, now, now])
    .zrange([key, 0, 0])
    .zrange([key, -max, -max])
    //.zremrangebyrank([key, 0, -(max + 1)])
    .pexpire([key, duration])
    .exec(function (err, res)

```

**Comment by noamshemesh** [comment]
can you please open a PR with relevant tests?

**Comment by penghap** [comment]
Yes, I have created a PR and passed tests, please review the change.

**Comment by noamshemesh** [comment]
merged #47
